src/clj_headlights/bigtable_io.clj

```diff
@@ -57,12 +57,16 @@
 
 (defn bigtable-io-write
   [pcoll name project {:keys [instance user-agent table]}]
-  (-> pcoll
-      (df/df-map (str "make-bigtable-KV-" name) #'row->bigtable-kv)
-      (.apply (str "write-to-bigtable-" name)
-              (-> ^BigtableIO$Write (BigtableIO/write)
-                  (.withBigtableOptions ^BigtableOptions$Builder (options-builder project instance user-agent))
-                  (.withTableId table)))))
+  (df/composite
+    name
+    [pcoll]
+    (fn [pcoll]
+      (-> pcoll
+          (df/df-map "make-bigtable-kv" #'row->bigtable-kv)
+          (.apply "write"
```

Comment:
Can you append the name parameter to these pcoll names?

Reply:
I removed it on purpose: it is already contained in the composite name. Why do you want it?

Reply:
Right, when they're nested you don't need to do this. Carry on.